Source/EmbeddedBoundary/WarpXFaceExtensions.cpp

```diff
@@ -164,6 +163,27 @@ ComputeNBorrowOneFaceExtension(const amrex::Dim3 cell, const amrex::Real S_ext,
             }
         }
     }else if(idim == 1){
+#ifdef WARPX_DIM_XZ
+        for (int i_n = -1; i_n < 2; i_n++) {
+            for (int j_n = -1; j_n < 2; j_n++) {
+                //This if makes sure that we don't visit the "diagonal neighbours"
+                if (!(i_n == j_n || i_n == -j_n)) {
+                    // Here a face is available if it doesn't need to be extended itself and if its
+                    // area exceeds Sz_ext. Here we need to take into account if the intruded face
+                    // has given away already some area, so we use Sz_red rather than Sz.
+                    // If no face is available we don't do anything and we will need to use the
+                    // multi-face extensions.
+                    if (S_red(i + i_n, j + j_n, k) > S_ext
+                        && (flag_info_face(i + i_n, j + j_n, k) == 1
+                             || flag_info_face(i + i_n, j + j_n, k) == 2)
+                        && flag_ext_face(i, j, k) && ! stop) {
+                        n_borrow += 1;
+                        stop = true;
+                    }
+                }
+            }
+        }
+#else
```

Comment:
Let's make this
```suggestion
#elif defined(WARPX_DIM_3D)
```
and then add an
```C++
#else
    amrex::Abort("CountExtFaces: Only implemented in 2D3V and 3D3V")
```
so that we catch at runtime:
- 1D3V
- RZ
until they are added later on.

Reply:
We actually also have another check at the beginning of WarpX::InitEB to abort in case EB and RZ are on at the same time. I can add a similar check for 1D3V there.
I can still add this check here because it makes the code more self-explanatory and less confusing.